arrebol/worker/auth/token/jwt.go

```diff
@@ -0,0 +1,80 @@
+package token
+
+import (
+	//...
+	// import the jwt-go library
+	"github.com/dgrijalva/jwt-go"
+	"github.com/ufcg-lsd/arrebol-pb/arrebol/worker"
+	"github.com/ufcg-lsd/arrebol-pb/crypto"
+	"log"
+	"os"
+	"time"
+
+	//...
+)
+
+const (
+	ArrebolPrivKeyPath = "ARREBOL_PRIV_KEY_PATH"
+	ArrebolPubKeyPath  = "ARREBOL_PUB_KEY_PATH"
+)
+
+type JWToken struct {
+	raw string
+	QueueId string
+	WorkerId string
+}
+
+type Claims struct {
+	QueueId string `json:"QueueId"`
+	WorkerId string `json:"WorkerId"`
+	jwt.StandardClaims
+}
+
+func NewJWToken(worker *worker.Worker) (*JWToken, error){
```

Comment:
please, add a doc explaining this func

Reply:
why you return a token with raw, queueId and workerId, if the raw already contains this information?

Reply:
I updated JWToken and these fields have been removed.